{
    "ConnectIOException.class": {
        "modified": "2006-12-12 10:41:51",
        "size": "0.63KB",
        "md5": "e0da54aaf82429a2c952b6ca4e45917d",
        "sha256": "1ae587e4e706b0f72d20946ac9ff4fc662825f7f142bb93b153760b6a934efbc"
    },
    "Metadata": {
        "Content-Type": "application/java-vm",
        "X-Parsed-By": [
            "org.apache.tika.parser.DefaultParser",
            "org.apache.tika.parser.asm.ClassParser"
        ],
        "X-TIKA:content_handler": "ToTextContentHandler",
        "X-TIKA:embedded_depth": "0",
        "X-TIKA:parse_time_millis": "2",
        "dc:title": "ConnectIOException",
        "resourceName": "ConnectIOException.class",
        "title": "ConnectIOException"
    }
}